### PR TITLE
gh-106976: alphabetise bullets by module name task2-3

### PR DESCRIPTION
**Pull request by littlebutt**
- Update the document [What's New in Python 3.12](https://docs.python.org/3.13/whatsnew/3.12.html)

* Issue: gh-106976

----
:books: Documentation preview :books:: https://cpython-previews--107005.org.readthedocs.build/

### CODE REVIEW

File: Doc/whatsnew/3.12.rst [hunk 1]
```diff
@@ -966,84 +966,83 @@ Demos and Tools
...
   Python 3.14, when ``'data'`` filter will become the default.
   See :ref:`tarfile-extraction-filter` for details.
 
-* ``calendar.January`` and ``calendar.February`` constants are deprecated and
-  replaced by :data:`calendar.Month.JANUARY` and :data:`calendar.Month.FEBRUARY`.
-  (Contributed by Prince Roshan in :gh:`103636`.)
+* :mod:`typing`: :class:`typing.Hashable` and :class:`typing.Sized` aliases for :class:`collections.abc.Hashable`
+  and :class:`collections.abc.Sized`. (:gh:`94309`.)
+
+* :mod:`xml`: The module now emits :exc:`DeprecationWarning`
```
Comment (hugovk) [inline Doc/whatsnew/3.12.rst]: Let's keep the full one here:

```suggestion
* :mod:`xml.etree.ElementTree`: The module now emits :exc:`DeprecationWarning`
```

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks, one small update needed.

**Comment by littlebutt** [comment]
I have made the requested changes; please review again

**Review by hugovk — APPROVED** [review]
Thanks again!

**Pull request merged**